src/containers/dashboard/DashboardContainer.js

```diff
@@ -1,204 +1,46 @@
-/*
- * @flow
- */
-
+// @flow
 import React from 'react';
 
-import styled from 'styled-components';
-import { Map } from 'immutable';
-import { Spinner } from 'lattice-ui-kit';
-import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
-import type { RequestSequence } from 'redux-reqseq';
-
-import { loadDashboardData } from './DashboardActionFactory';
-import { SUBMISSION_STATES } from './DashboardReducer';
-
-import ButtonToolbar from '../../components/buttons/ButtonToolbar';
-import DropdownButton from '../../components/buttons/DropdownButton';
-import IncidentCharts from '../../components/dashboard/IncidentCharts';
-import OutcomeCharts from '../../components/dashboard/OutcomeCharts';
-import OverviewCharts from '../../components/dashboard/OverviewCharts';
-import SummaryStats from '../../components/dashboard/SummaryStats';
-import { SUMMARY_STATS } from '../../shared/Consts';
-
-const OptionRow = styled.div`
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  align-items: center;
-`;
-
-const Flex = styled.div`
-  display: flex;
-  flex: 1;
-  justify-content: center;
-  align-items: center;
-  width: 100%;
-`;
-
-const DashboardWrapper = styled.div`
-  width: 100%;
-`;
-
-const NoReports = styled.div`
-  margin-top: 50px;
-  width: 100%;
-  text-align: center;
-  font-size: 16px;
-  color: #2e2e34;
-`;
-
-const LAYOUTS = {
-  OVERVIEW: 'Overview',
-  INCIDENT: 'Incident',
-  OUTCOMES: 'Outcomes'
-};
-
-const TIME_LENGTHS = {
-  MONTHS_1: {
-    label: '1 Month',
-    value: 1
-  },
-  MONTHS_3: {
-    label: '3 Months',
-    value: 3
-  },
-  MONTHS_6: {
-    label: '6 Months',
-    value: 6
-  },
-  YEAR: {
-    label: '1 Year',
-    value: 12
+import { DashboardContainer as CRCDashboardContainer } from '@openlattice/lattice-community-restorative-court';
+import { useSelector } from 'react-redux';
+import { useRouteMatch } from 'react-router';
+
+import CrisisReportDashboardContainer from './CrisisReportDashboardContainer';
+
+import { useAppSettings } from '../../components/hooks';
+import { ContentOuterWrapper, ContentWrapper } from '../../components/layout';
+import { DASHBOARD_PATH } from '../../core/router/Routes';
+
+const DashboardContainer = () => {
+  const settings = useAppSettings();
+  const profileModule = settings.get('profileModule', 'crisis');
+  const organizationId = useSelector((state) => state.getIn(['app', 'selectedOrganizationId']));
+  const match = useRouteMatch();
+
+  let component = null;
+
+  switch (profileModule) {
+    case 'crisis':
+      component = <CrisisReportDashboardContainer />;
+      break;
+    case 'crc':
+      component = <CRCDashboardContainer root={DASHBOARD_PATH} match={match} organizationId={organizationId} />;
+      break;
+    case 'helpline':
+      component = <CrisisReportDashboardContainer />;
+      break;
+    default:
+      component = <CrisisReportDashboardContainer />;
```

Comment:
Would have been nice to constantify these strings

Looks like this switch statement only needs two cases: `crc` and `default`